demos/smart_classroom_demo/README.md

```diff
@@ -122,9 +105,8 @@ Example of a valid command line to run the application for recognizing first rai
 
 The demo uses OpenCV to display the resulting frame with labeled actions and faces.
 
-> **NOTE**: On VPU devices (Intel® Movidius™ Neural Compute Stick, Intel® Neural Compute Stick 2, and Intel® Vision Accelerator Design with Intel® Movidius™ VPUs) this demo has been tested on the following Model Downloader available topologies: 
+> **NOTE**: On VPU devices (Intel® Movidius™ Neural Compute Stick, Intel® Neural Compute Stick 2, and Intel® Vision Accelerator Design with Intel® Movidius™ VPUs) this demo has been tested on the following Model Downloader available topologies:
 >* `face-detection-adas-0001`
->* `face-reidentification-retail-0095`
 >* `landmarks-regression-retail-0009`
```

Comment:
seems with removing -m_lm option this also should be removed?

Reply:
I didn't remove the option.